### PR TITLE
Set a fixed width to the default columns on each selected field

### PR DESCRIPTION
**Pull request by guidomodarelli**
### Description
Set a fixed width to the default columns on each selected field
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/7059

### Evidence
<details>
  <summary>AWS</summary>

image

</details>
<details>
  <summary>Docker</summary>

image

</details>

<details>
  <summary>FIM</summary>

image

</details>

<details>
  <summary>GDPR</summary>

image

</details>

<details>
  <summary>Github</summary>

image

</details>

<details>
  <summary>Google Cloud</summary>

image

</details>

<details>
  <summary>HIPAA</summary>

image

</details>

<details>
  <summary>Malware Detection</summary>

image

</details>

<details>
  <summary>MITRE ATT&CK</summary>

image

</details>

<details>
  <summary>NIST</summary>

image

</details>

<details>
  <summary>Office 365</summary>

image

</details>

<details>
  <summary>PCI DSS</summary>

image

</details>

<details>
  <summary>Threat Hunting</summary>

image

</details>

<details>
  <summary>TSC</summary>

image

</details>

<details>
  <summary>Vulnerabilities Detection</summary>

image

</details>

#### 

### Test
1. Navigate to the Events panel for each Application.
2. Verify that the columns have a fixed width for the small columns and allow the large column to expand.

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by JuanGarriuz** [comment]
### Test

<details>
  <summary>AWS 🟢 </summary>

image


</details>
<details>
  <summary>Docker 🟢</summary>

image


</details>

<details>
  <summary>FIM 🟢</summary>

image


</details>

<details>
  <summary>GDPR 🟢</summary>

image


</details>

<details>
  <summary>Github 🟢</summary>

image


</details>

<details>
  <summary>Google Cloud 🟢</summary>

image

</details>

<details>
  <summary>HIPAA 🟢</summary>

image


</details>

<details>
  <summary>Malware Detection 🟢</summary>

image

</details>

<details>
  <summary>MITRE ATT&CK 🟢</summary>

image


</details>

<details>
  <summary>NIST 🟢</summary>

image

</details>

<details>
  <summary>Office 365 🟢</summary>

image

</details>

<details>
  <summary>PCI DSS 🟢</summary>

image

</details>

<details>
  <summary>Threat Hunting 🔴 </summary>

image

</details>

<details>
  <summary>TSC 🟢</summary>

image

</details>

<details>
  <summary>Vulnerabilities Detection 🟢</summary>

image

</details>

File: plugins/main/public/components/overview/threat-hunting/events/threat-hunting-columns.tsx
Comment (JuanGarriuz) [inline plugins/main/public/components/overview/threat-hunting/events/threat-hunting-columns.tsx:1]: Event's page is defined in

`plugins/main/public/components/common/wazuh-discover/config/data-grid-columns.tsx`

image

`import { threatHuntingColumns } from '../wazuh-discover/config/data-grid-columns';`

**Review by JuanGarriuz — APPROVED** [review]
### Test

<details>
  <summary>AWS 🟢 </summary>

image


</details>
<details>
  <summary>Docker 🟢</summary>

image


</details>

<details>
  <summary>FIM 🟢</summary>

image


</details>

<details>
  <summary>GDPR 🟢</summary>

image


</details>

<details>
  <summary>Github 🟢</summary>

image


</details>

<details>
  <summary>Google Cloud 🟢</summary>

image

</details>

<details>
  <summary>HIPAA 🟢</summary>

image


</details>

<details>
  <summary>Malware Detection 🟢</summary>

image

</details>

<details>
  <summary>MITRE ATT&CK 🟢</summary>

image


</details>

<details>
  <summary>NIST 🟢</summary>

image

</details>

<details>
  <summary>Office 365 🟢</summary>

image

</details>

<details>
  <summary>PCI DSS 🟢</summary>

image

</details>

<details>
  <summary>Threat Hunting 🟢  </summary>

image

</details>

<details>
  <summary>TSC 🟢</summary>

image

</details>

<details>
  <summary>Vulnerabilities Detection 🟢</summary>

image

</details>


### CR 🟢 

LGTM!!

**Review by chantal-kelm — APPROVED** [review]
### Evidence
<details>
  <summary>AWS</summary>
Captura de pantalla 2024-10-14 a la(s) 5 10 58 p  m



</details>
<details>
  <summary>Docker</summary>

Captura de pantalla 2024-10-14 a la(s) 5 11 53 p  m


</details>

<details>
  <summary>FIM</summary>
Captura de pantalla 2024-10-14 a la(s) 5 12 04 p  m



</details>

<details>
  <summary>GDPR</summary>

Captura de pantalla 2024-10-14 a la(s) 5 12 24 p  m


</details>

<details>
  <summary>Github</summary>
Captura de pantalla 2024-10-14 a la(s) 5 12 50 p  m



</details>

<details>
  <summary>Google Cloud</summary>
Captura de pantalla 2024-10-14 a la(s) 5 13 17 p  m



</details>

<details>
  <summary>HIPAA</summary>

Captura de pantalla 2024-10-14 a la(s) 5 14 08 p  m


</details>

<details>
  <summary>Malware Detection</summary>
Captura de pantalla 2024-10-14 a la(s) 5 14 25 p  m



</details>

<details>
  <summary>MITRE ATT&CK</summary>

Captura de pantalla 2024-10-14 a la(s) 5 14 45 p  m


</details>

<details>
  <summary>NIST</summary>


Captura de pantalla 2024-10-14 a la(s) 5 15 29 p  m

</details>

<details>
  <summary>Office 365</summary>

Captura de pantalla 2024-10-14 a la(s) 5 15 52 p  m


</details>

<details>
  <summary>PCI DSS</summary>

Captura de pantalla 2024-10-14 a la(s) 5 16 13 p  m


</details>

<details>
  <summary>Threat Hunting</summary>

Captura de pantalla 2024-10-14 a la(s) 5 17 43 p  m


</details>

<details>
  <summary>TSC</summary>
Captura de pantalla 2024-10-14 a la(s) 5 18 27 p  m



</details>

<details>
  <summary>Vulnerabilities Detection</summary>

Captura de pantalla 2024-10-14 a la(s) 5 18 47 p  m


</details>

**Review by chantal-kelm — APPROVED** [review]

**Review by Machi3mfl — APPROVED** [review]
### Test

<details>
  <summary>Malware Detection, FIM, Threat Hunting, Vulnerability Detection, Mitre Attack 🟢 </summary>

https://github.com/user-attachments/assets/3f3c8408-c4ca-496b-803e-07791d65b47e


</details>

<details>
  <summary>PCI, GDPR, HIPAA, NIST, TSC 🟢</summary>

https://github.com/user-attachments/assets/42af197e-ef16-4f78-9c06-2887e3669fe6


</details>

<details>
  <summary> Docker, AWS, Google Cloud, GitHub, Office 365 🟢</summary>

https://github.com/user-attachments/assets/6c6a3191-ddaa-470b-9b1f-786252f358cd


</details>

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**